Woof/Woof/View/OwnerProfileView/OwnerProfileView.swift

```diff
@@ -0,0 +1,55 @@
+import SwiftUI
+
+/// View which could be used to display or edit information about Owner
+struct OwnerProfileView: View {
+    /// View model responsible to manage data from model layer
+    @ObservedObject var viewModel = OwnerProfileViewModel()
+
+    /// Owner information editing status tracker
+    @State private var isEditingTracker = false
```

Comment:
nit:
```suggestion
    /// Owner information editing status tracker
    @State private var isEditingMode = false
```
or 
```suggestion
    /// Owner information editing status tracker
    @State private var isViewEditung = false
```

Reply:
@NadzeyaShpakouskaya maybe you meant:
@State private var `isViewEditing` = false

Reply:
i'll choose the first one option